docs/getting-started-guides/locally.md

```diff
@@ -129,6 +132,19 @@ cluster/kubectl.sh create -f docs/user-guide/pod.yaml
 
 Congratulations!
 
+### Hacking hack-local-up cluster to test more sophisticated deployments, such as HA.
+
+To test highly available apiservers, you can modify local-up-cluster.sh as follows.
+
+1) Write a kubernetes token file which has a kubelet user with a token defined.  Lets call it /tmp/mytokens for concreteness.
+
+2) Launch the apiserver with the `--token-auth-file=/tmp/mytokens` and `--apiserver-node-port=30279` options.  Of course 30279 is just an example port.
```

Comment:
Ok just to be explicit , I think youre suggesting that we can have a loadbalancer on top of the nodeports so that if you lose a nodeport on a kubelet others can be used.  Which is something that also is important I guess. Beeps takes care of this by having a node label=loadbalancer so that some kubelets specifically work as load balancers.  We also will be testing that in the next few days